tests/test_helpers.py

```diff
@@ -0,0 +1,94 @@
+import os
+import shutil
+
+import pytest
+import torch
+
+import kaolin as kal
+from kaolin import helpers
+
+
+CACHE_DIR = 'tests/cache'
+
+
+@pytest.fixture(autouse=True)
+def cleanup():
+    """Cleanup after each test. """
+    yield
+    shutil.rmtree(CACHE_DIR)
+
+
```

Comment:
Can we test also the correctness with a dictionary of mixed Tensor / numpy.array ?

Reply:
Added a numpy array to the dictionary test.